opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/opennms-graph.properties

```diff
@@ -38,7 +38,10 @@ OpenNMS.Newts.Index.Inserts, \
 OpenNMS.Newts.Index.Update.Latency, \
 OpenNMS.Newts.Index.Delete.Latency, \
 OpenNMS.Cassandra.Cluster1.Requests, \
-OpenNMS.Cassandra.Cluster1.Request.Latency
+OpenNMS.Cassandra.Cluster1.Request.Latency, \
+metrics.OpenNMS.Sink.Heartbeat.HearbeatLag.AttributeReport, \
```

Comment:
Can we rename these reports to `OpenNMS.Sink.Heartbeat.Lag` and `OpenNMS.Sink.Trap.Lag` instead?